exercises/list-ops/canonical-data.json

```diff
@@ -223,6 +223,14 @@
             "list": [1, 3, 5, 7]
           },
           "expected": [7, 5, 3, 1]
+        },
+        {
+          "description": "list of nested lists",
```

Comment:
I dispute any assertion that the input is a list of nested lists, because each element of the list is not a nested list (there are no additional levels of lists in each element of the input list). So it seems that `list of lists` is a better name.

I also refer to the standard https://github.com/exercism/problem-specifications/blob/master/README.md#test-data-format-canonical-datajson "the description should not simply explain **what** each case is (that is redundant information) but also **why** each case is there." , and suggest that the description mention something about not flattening or reversing any inner lists.

Reply:
Will `list of lists is not flattened` suffice?

Reply:
:+1: